src/components/BadgerBadge/BadgerBadge.js

```diff
@@ -168,6 +171,7 @@ class BadgerBadge extends React.PureComponent<Props> {
 								symbol={coinSymbol}
 								name={coinName}
 								paymentRequestUrl={paymentRequestUrl}
+								invoiceTimeLeftSeconds={invoiceTimeLeftSeconds}
```

Comment:
Not liking how this flow is working.  
Right now it says "BIP70 invoice" where `amount` goes, and a timer where the `fiat` amount would go. and the timer is passed in twice, when only displayed once...
This doesn't make sense.  


Perhaps for BIP70 it should render a new section for `BIP 70 invoice` and if `showAmount` is enabled, then show the BCH and fiat values.  
If this flow isn't addressed now it will become technical debt in the future.

Reply:
cd355b4